src/runtime/ModuleStore.js

```diff
@@ -152,9 +152,12 @@
               evaled.push(frame);
             }
           });
-          ex.stack = evaled.join('\n');
+          // Allocate new object, some platforms make ex.stack read-only.
+          ex = {
```

Comment:
This seems strange. Why do we want to lose the exception object here.

Could you elaborate with a comment why we need to change the stack property in the first place?

Is the property configurable? Maybe use defineProperty or if not I would prefer that we skip updating it if the stack property is non configurable.